views/login.hbs

```diff
@@ -0,0 +1,7 @@
+<form action="/Login" method="post">
+    <label for="username">Username</label>
```

Comment:
Pensez que vous pouvez aussi mettre l'`<input>` dans le `<label>` et vous passer alors de `for=""`/`id=""`